### ISSUE TITLE
i18n support for arch wiki

### ISSUE THREAD

**Issue by sainnhe** [filiparag/wikiman#3]
Looks like this script search for English only arch wiki [L53](https://github.com/filiparag/wikiman/blob/a54402d08a2d63ff1467b08f210f59cbb10ba23c/wikiman.sh#L53)

Could you add an option to specify the language to search for? For example:

```shell
wikiman --lang=zh_CN bootctl
```

**Comment by filiparag** [comment]
Support for multiple languages added in version [2.1](https://github.com/filiparag/wikiman/releases/tag/2.1).

To add Chinese, set user configuration file `~/.config/wikiman/wikiman.conf` accordingly:
```ini
# Manpages language
man_lang = en, zh_CN

# ArchWiki language
wiki_lang = en, zh-CN
```

Support for POSIX-style flags is planned for a future release.

One issue that I couldn't solve is forcing `apropos -L zh_CN <search>` to only return man pages available in target locale without falling back to English. If you know how to do it, please let me know.

**Comment by sainnhe** [comment]
Thank you!

---

> One issue that I couldn't solve is forcing apropos -L zh_CN <search> to only return man pages available in target locale without falling back to English. If you know how to do it, please let me know.

I tried to use the `-M` flag instead of `-L` and it works fine, but I'm not sure if it will produce other bugs:

```
$ apropos -M /usr/share/man/zh_CN man
accessdb (8)         - 以易读的格式转储 man-db 数据库的内容
catman (8)           - 创建或更新预格式化的手册页
lexgrog (1)          - 解析 man 手册页的头部信息
man (1)              - 系统参考手册的接口
man-recode (1)       - 将手册页转换为另一种编码
manconv (1)          - 将手册页从一种编码转换到另一种
mandb (8)            - 创建或更新手册页索引缓存
manpath (1)          - 确定手册页的搜索路径
manpath (5)          - 格式化 /etc/man_db.conf 文件
```

**Comment by filiparag** [comment]
Thanks for pointing out `-M` flag. I saw it before already, but didn't think enough outside the box to use it.
Issue resolved in version [2.5](https://github.com/filiparag/wikiman/releases/tag/2.5)
